package.json

```diff
@@ -181,11 +181,12 @@
     "spectron": "brave/spectron#chromium60",
     "standard": "9.0.0",
     "style-loader": "^0.13.0",
-    "uglify-js-harmony": "^2.7.5",
+    "uglify-js": "git://github.com/mishoo/UglifyJS2.git#harmony-v2.8.22",
+    "uglifyjs-webpack-plugin": "^0.4.6",
     "uuid": "^3.0.1",
     "webdriverio": "4.7.1",
-    "webpack": "^1.12.9",
-    "webpack-dev-server": "^1.14.0",
+    "webpack": "^3.4.1",
+    "webpack-dev-server": "^2.6.1",
```

Comment:
sgtm. if we follow the development and make sure that failures which can be introduced by future minor updates can be fixed easily, let's change the tildes to carets :-)